src/state/wallets/walletsStore.ts

```diff
@@ -552,7 +569,7 @@ async function refreshAccountInfo(account: RainbowAccount, useCachedENS = false)
       ...account,
       image: newImage,
       // always prefer our label
-      label: account.label || ens,
+      label: account.label ? label : ens,
```

Comment:
nice catch. could probably do `label || ens`, but good with this as well as it's fundamentally no different